src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -828,11 +821,24 @@ public void removeBuilding(Building building)
         ColonyManager.markDirty();
     }
 
-    /*
-     *
-     * CITIZENS
-     *
+    /**
+     * Getter which checks if jobs should be manually allocated.
+     * @return true of false.
      */
+    public boolean isManualHiring()
+    {
+        return manualHiring;
+    }
+
+    /**
+     * Setter to set the job allocation manual or automatic.
+     * @param manualHiring true if manual, false if automatic.
+     */
+    public void setManualHiring(boolean manualHiring)
+    {
+        this.manualHiring = manualHiring;
+        markDirty();
+    }
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)